mixer/pkg/api/grpcServer.go

```diff
@@ -230,6 +367,83 @@ func (s *grpcServer) check(ctx context.Context, req *mixerpb.CheckRequest,
 	return resp, nil
 }
 
+
+//Access log service shold return empty response
+func (s *grpcServerEnvoy) StreamAccessLogs(
+	srv accessLogGRPC.AccessLogService_StreamAccessLogsServer) (error) {
+	for {
+		ctx := context.TODO()
+
+		msg, err := srv.Recv()
+		if err != nil {
+			return err
+		}
+
+		var totalBags int
+		if httpLogs := msg.GetHttpLogs(); httpLogs != nil {
+			totalBags = len(httpLogs.GetLogEntry())
+		} else if tcpLogs := msg.GetTcpLogs(); tcpLogs != nil {
+			totalBags = len(tcpLogs.GetLogEntry())
+		}
+
+		if s.throttler.Throttle(loadshedding.RequestInfo{PredictedCost: float64(totalBags)}) {
+			return grpc.Errorf(codes.Unavailable, "Server is currently overloaded. Please try again.")
+		}
+
+		reportSpan, reportCtx := opentracing.StartSpanFromContext(ctx, "Report")
```

Comment:
nit: It doesn't matter much, but I'd use something other than `Report`, to avoid confusion (here and below). Maybe `StreamAccessLogsReport` ?

Reply:
I'll  make that change.